policyengine_us/variables/gov/states/al/tax/income/exemptions/retirement/al_retirement_exemption_eligible_person.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class al_retirement_exemption_eligible_person(Variable):
```

Comment:
make three variables:
`al_retirement_exemption_eligible_person`
- the return here should be `eligible_age & head_or_spouse`
`al_retirement_exemption_person`
- person level min_(pension_income, p.max_amount) and defined for eligible person
`al_retirement_exemption`
- same as it is now